### ISSUE TITLE
Large number of uploaded files cause index limits

### ISSUE THREAD

**Issue by jaegeral** [google/timesketch#1533]
Running the latest docker-dev, uploading a large number (>100) .plaso files to the same sketch is causing elastic to throw exceptions:

```[2020-12-16 11:58:33 +0000] [6774] [ERROR] Error handling request /api/v1/sketches/1/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.8/dist-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 269, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 38, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 269, in error_router
    return original_handler(e)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/_compat.py", line 38, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/usr/local/src/timesketch/timesketch/api/v1/resources/sketch.py", line 237, in get
    es_stats = self.datastore.client.indices.stats(
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/client/utils.py", line 84, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/client/indices.py", line 769, in stats
    return self.transport.perform_request(
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/transport.py", line 351, in perform_request
    status, headers_response, data = connection.perform_request(
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/connection/http_urllib3.py", line 261, in perform_request
    self._raise_error(response.status, raw_data)
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/connection/base.py", line 181, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(
elasticsearch.exceptions.RequestError: RequestError(400, 'too_long_frame_exception', 'An HTTP line is larger than 4096 bytes.')
[2020-12-16 11:58:34,037] timesketch.elasticsearch/ERROR Unable to count indexes (index not found)
Traceback (most recent call last):
  File "/usr/local/src/timesketch/timesketch/lib/datastores/elastic.py", line 602, in count
    result = self.client.count(index=indices)
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/client/utils.py", line 84, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/client/__init__.py", line 528, in count
    return self.transport.perform_request(
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/transport.py", line 351, in perform_request
    status, headers_response, data = connection.perform_request(
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/connection/http_urllib3.py", line 261, in perform_request
    self._raise_error(response.status, raw_data)
  File "/usr/local/lib/python3.8/dist-packages/elasticsearch/connection/base.py", line 181, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(
elasticsearch.exceptions.RequestError: RequestError(400, 'too_long_frame_exception', 'An HTTP line is larger than 4096 bytes.')
```

To reproduce:

Create plaso files based on https://github.com/sbousseaden/EVTX-ATTACK-SAMPLES
and then run (used in a notebook here):
```
from picatrix import notebook_init
import os

notebook_init.init()

%timesketch_set_active_sketch 1

rootdir = "./single_events_plaso/"

for subdir, dirs, files in os.walk(rootdir):
    for file in files:
        #print os.path.join(subdir, file)
        filepath = subdir + os.sep + file

        if filepath.endswith(".plaso"):
            %timesketch_upload_file $filepath
            print (filepath)
```

Expected behaviour: 
- better error handling if it occurs
- or even better, avoid so many uploads to a sketch and give a warning, maybe set an upper limit of 50? 100?

**Comment by jaegeral** [comment]
There is a related thing: https://github.com/olivere/elastic/issues/1215

**Comment by kiddinn** [comment]
so a qq for you regarding this, what version of ES are you running on your dev docker?

the bug you refer to seems to suggest a v6 edition issue, with this fixed in v7.... wondering if this is still an issue in later ES stacks?

however I will add some error handling to the codebase

**Comment by kiddinn** [comment]
ok, with #1539 there were some changes made to this code, and things are working slightly differently.

@jaegeral can you do two things:
1. Update to the latest code
2. Check your ES version and update to 7.x (you may need to do `sudo docker-compose pull elasticsearch` to get a new version, then to stop/`up -d` the instance (and re-create all indices since that will destroy the ES store)

And then to attempt to reproduce this issue and let me know if htis is still an issue, and how it presents itself in the latest version

**Comment by kiddinn** [comment]
This should be fixed by #1567 , now all data in a timeline shares the same index.

Can you test again using the newest version, to see if we can close this issue down?
